app/lib/frontend/dom/dom.dart

```diff
@@ -399,6 +399,32 @@ Node li({
       children: _children(children, child, text),
     );
 
+/// Creates an `<option>` Element using the default [DomContext].
+Node option({
+  String? id,
+  Iterable<String>? classes,
+  Map<String, String>? attributes,
+  Iterable<Node>? children,
+  Node? child,
```

Comment:
Does it ever make sense to have children or child, isn't `text` the only thing worth using?

Reply:
I think material and other frameworks allow the creation of complex selects, with icons or formatted content. Not sure if they need rich content or data attributes for that, but let's keep that open.